samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb

```diff
@@ -30,7 +30,7 @@
   },
```

Comment:
<p>Map doesn't take zoom as a parameter. You can move this to a cell below and do <code>m.zoom = 9</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>